src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -555,6 +555,28 @@ paths:
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
 
+  /state/validators/uptime:
```

Comment:
I think validator update should not be part of `state/*` endpoint group as it has nothing to do with the ledger state? @dhedey ?

Reply:
So long as it takes a state version and is deterministic against that state version, I'm happy with it being in state.

But also happy with it being in a different place, eg `/aggregate/` or `/statistics/` or `/data/` or something.